lib/compiler/src/beam_core_to_ssa.erl

```diff
@@ -93,7 +93,7 @@
 %% matching. (Construction of those term types is translated directly
 %% to SSA instructions.)
 
--record(cg_tuple, {es}).
+-record(cg_tuple, {es,keep=[]}).
```

Comment:
Nitpick: this is always used as an ordered set. Perhaps make that explicit here?

```suggestion
-record(cg_tuple, {es,keep=ordsets:new()}).
```